spectrum/checks.py

```diff
@@ -881,6 +881,12 @@ def _build_url(path, host):
     'articles/{id}/elife-{id}-v{version}.pdf',
     'articles/{id}/elife-{id}-v{version}.pdf'
 )
+DIGEST_JPG_PUBLISHED_CDN_BUCKET = BucketFileCheck(
+    aws.S3,
+    SETTINGS['bucket_published'],
+    'digests/{id}/digest-{id}.jpg',
```

Comment:
All looks good, except this new file naming isn't deposited just yet. It shouldn't be too hard to add very soon.

Reply:
Yes, commented out for the time being